packages/react-router-native/jest.config.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
   preset: "react-native",
+  rootDir: "../..",
   testRunner: "jest-circus/runner",
+  testMatch: ["<rootDir>/packages/react-router-native/__tests__/*.js"],
```

Comment:
The React Native Jest preset is weird and I have to set the root as the `rootDir` to get it to work, as well as set the `testMatch` so that it doesn't match every test found from the root.